conduct/artifacts/CODE_OF_CONDUCT_PREFIX.md

```diff
@@ -0,0 +1,44 @@
+# Code of Conduct
+
+The OpenJS Foundation and its member projects use Contributor Covenant as their code of conduct. The full text is included below in English, and [translations](https://www.contributor-covenant.org/translations) are available on the Contributor Covenant website.
+
+## Commitment
+
+All recipients of reports commit to:
+
+- maintain the confidentiality with regard to the reporter and victim of an incident, and
+- participate in the path for escalation as outlined in the section on Escalation when required.
```

Comment:
this should either be a paragraph or bullets, not both together. Fine with either suggestion or alternatives: 

```suggestion
All recipients of reports commit to:

- maintain the confidentiality with regard to the reporter and victim of an incident
- participate in the path for escalation as outlined in the section on Escalation when required
```


```suggestion
All recipients of reports commit to maintain the confidentiality with regard to the reporter and victim of an incident, and participate in the path for escalation as outlined in the section on Escalation when required.
```

Reply:
Bulets make it easier to add to should such adding be required.

Reply:
Should we make these changes after the initial document is passed, and merged? I think it will reduce friction and will make the next steps easier. Depending on preference, I can make the suggestions here as an additional commit.

Reply:
I just added a commit to address this issue. Appreciate if you can review it, thank you.